addon/components/bs-form.js

```diff
@@ -154,18 +155,6 @@ export default Ember.Component.extend({
   novalidate: false,
 
   /**
-   * An array of `Components.FormElement`s that are children of this form.
-   *
-   * @property childFormElements
-   * @type Array
-   * @readonly
-   * @protected
-   */
-  childFormElements: computed.filter('childViews', function(view) {
```

Comment:
That makes sence. Updated PR. I guess commits should be squashed before merge...